src/main/groovy/de/triplet/gradle/play/PlayPublishListingTask.groovy

```diff
@@ -41,7 +41,7 @@ class PlayPublishListingTask extends PlayPublishTask {
         super.publish()
 
         // Matches if locale have the correct naming e.g. en-US for play store
-        inputFolder.eachDirMatch(matcher) { dir ->
+        if (inputFolder.exists()) inputFolder.eachDirMatch(matcher) { dir ->
```

Comment:
In this case, the following statements from line 131 would be executed. What do you think about "inverting the flow" by
```
if (!inputFolder.exists()) {
    return
}
```